platform/features/conductor-redux/src/README.md

```diff
@@ -0,0 +1,187 @@
+5. Switch to follow time. Set inner end to now, and outer end to now + 1 hour
+6. Plot controller updated on tick
+7. Plot Controller updated when user changes bounds
+8. Conductor controller needs to update bounds and mode on TC when user changes bounds
+
+### Additional possible use-cases
+1. Telemetry adapter wants to indicate presence of data at a particular time
+2. Time conductor controller wants to paint map of data availability.
+
+These use-cases could be features of the TimeConductor, but perhaps makes 
+sense to make knowledge of data availability the sole preserve of telemetry 
+adapters, not TimeConductor itself. Adapters will be ultimately responsible 
+for providing these data so doesn't make much sense to duplicate elsewhere.
+The TimeConductorController - which knows tick interval on scale (which 
+TimeConductor API does not) - could simply request data availability from 
+telemetry API and paint it into the Time Conductor UI
+
+## Example implementations of use cases
+### 1. Historical session is loaded and system sets time bounds on conductor
+``` javascript
+function loadSession(session) {
+    var tc = MCT.conductor;
+    tc.timeSystem(session.metadata.timeSystem);
+    
+    //Set start and end modes to fixed date
+    tc.startMode(tc.Modes.FIXED);
+    tc.endMode(tc.Modes.FIXED);
+    
+    //Set both inner and outer bounds
+    tc.bounds.inner.start.time(session.metadata.firstTime);
+    tc.bounds.inner.end.time(session.metadata.lastTime);
+    tc.bounds.outer.start.time(session.metadata.firstTime);
+    tc.bounds.outer.end.time(session.metadata.lastTime);
+}
+```
+### 2. User changes time of interest
+```javascript
+//Somewhere in the TimeConductorController...
+function changeTOI(newTime) {
+    MCT.conductor.timeOfInterest(newTime);
+}
+```
+### 3. Plot controller listens for change to TOI
+```javascript
+// toi is attribute of Time Conductor object. Add a listener to the time 
+// conductor to be alerted to changes in value
+
+MCT.conductor.listen(function (conductor) {
+    var toi = conductor.timeOfInterest();
+    plot.setTimeOfInterest(toi);
+});
+```
+### 4. Real-time object is loaded and time conductor is set up in real-time mode showing last 15 minutes of data.
+```javascript
+// Notional initialization function, perhaps in a 'runs' extension
+function initializeConductor(telemetryMetadata) {
+    var tc = MCT.conductor,
+        FIFTEEN_MINUTES = 15 * 60 * 1000;
+    tc.timeSystem(telemetryMetadata.timeSystem);
+    
+    //Start and end modes set separately (to support fixed start, follow 
+    // time end. Follow time = RELATIVE end mode where delta value = 0) 
+    tc.startMode(tc.Modes.RELATIVE);
+    tc.endMode(tc.Modes.RELATIVE);
+    
+    // Delta defined in ms. Start time delta is relative to end time.
+    tc.bounds.inner.start.delta(FIFTEEN_MINUTES);
+    tc.bounds.outer.start.delta(FIFTEEN_MINUTES);
```

Comment:
I think this `delta` parameter could be removed, and the responsibility for converting from deltas to bounds delegated to the Mode:

```
tc.timeSystem(telemetryMetadata.timeSystem);
tcMode.switch(new FollowMode(FIFTEEN_MINUTES));
```

Again, idea here is to simplify public API - no need to worry about which mode you're in, you just have the ability to observe the parameters (start, end, TOI, time sys) which will effect queries or display elsewhere in the UI.